src/apps/rDSN.monitor/app_package/static/js/service_meta.js

```diff
@@ -59,26 +59,29 @@ var vm = new Vue({
                                 if(par.primary!='invalid address')
                                 {
                                     par.membership += 'P: ("' + par.primary + '"),\n ';
```

Comment:
primary maybe invalid while secondaries are ok, you cannot use this to check it is a stateful or stateless service